diff -urN bigboard-0.5.33/scripts/bigboard.in bigboard-0.5.33.new/scripts/bigboard.in
--- bigboard-0.5.33/scripts/bigboard.in	2008-04-23 19:38:32.000000000 +0200
+++ bigboard-0.5.33.new/scripts/bigboard.in	2008-04-26 18:11:12.000000000 +0200
@@ -1,4 +1,4 @@
 #!/bin/sh
 PYTHONPATH=$PYTHONPATH:@pyexecdir@/bigboard_externals
 export PYTHONPATH
-exec $BB_DEBUG_CMD @PYTHON@ @BB_PYDIR@/main.py "--stockdirs=@BB_STOCKDIR@" "$@"
+exec $BB_DEBUG_CMD @PYTHON@ @BB_PYDIR@/main.pyc "--stockdirs=@BB_STOCKDIR@" "$@"
